src/gmp/models/task.js

```diff
@@ -84,6 +85,7 @@ const TASK_STATUS_TRANSLATIONS = {
   Done: _l('Done'),
   Queued: _l('Queued'),
   Processing: _l('Processing'),
+  'Uploading Interrupted': _l('Interrupted'),
```

Comment:
Is that correct? Shouldn't it be
```suggestion
  'Uploading Interrupted': _l('Uploading Interrupted'),
```